testdata/local/pods.json

```diff
@@ -0,0 +1,2 @@
+{"_id":"localhost", "secret":"de249183582947721fdfb2ea1796574b","agent_revision":"foo", "type": "agent", "status": "running", "task_creation_opts": {"env_secrets": {"POD_SECRET": "de249183582947721fdfb2ea1796574b"}}}
```

Comment:
* Is `secret` necessary here? Seems like this just needs the `env_secrets`. Also [`env_secrets`](https://github.com/evergreen-ci/evergreen/blob/a822913f1b3cb5e2977e7d00866b8f289febdecf/model/pod/pod.go#L276-L279) seems to have a different structure than what's given here.
* * `agent_revision` seems like it should be `agent_version`?

Reply:
Yeah, the local `pods.json` deviated from the smoke `pods.json` as time / debugging progress went on, and didn't notice because I never needed to load this data locally. I'll make this file look like the one in the `smoke` folder, which is formatted correctly.